test/test_nestedtensor.py

```diff
@@ -5231,16 +5227,12 @@ def test_jagged_layout_construction_as_nested_tensor(
             expected_batch_size = len(tensor_list)
             expected_contiguous = True
             expected_min_seqlen = min(
-                [
-                    (torch.tensor(t) if isinstance(t, list) else t).shape[0]
-                    for t in tensor_list
-                ]
+                (torch.tensor(t) if isinstance(t, list) else t).shape[0]
```

Comment:
I think `torch.as_tensor(t)` would have this exact same semantics and simpler code at the same time

Reply:
Fair, but this is a test case

Reply:
Yeah, it should ideally be applied to all occurrences...

Reply:
Makes sense for a follow up PR